github/github/pr-reviewer/rust/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "pr-reviewer"
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+path = "src/pr-reviewer.rs"
+crate-type = ["cdylib"]
+
+[dependencies]
+# flows-connector-dsi = { version = "0.1" }
```

Comment:
Please use `0.1` to match the latest version of flows-connector-dsi

Reply:
🙆